db/seeds.rb

```diff
@@ -5,3 +5,19 @@
 unless User.exists?(email: "swillhide@interexchange.org")
   User.create!(name: "Stephanie Willhide", email: "swillhide@interexchange.org", password: "password")
 end
+
+unless Book.exists?(isbn: "0393312836")
+  Book.create!(name: "A Clockwork Orange", isbn: "0393312836")
+end
+
+unless Book.exists?(isbn: "0679734503")
+  Book.create!(name: "Crime and Punishment", isbn: "0679734503")
+end
+
+unless Book.exists?(isbn: "0321410971")
+  Book.create!(name: "Transcending CSS", isbn: "0321410971")
+end
+
+unless Book.exists?(isbn: "0399226907")
+  Book.create!(name: "The Very Hungry Caterpillar", isbn: "0399226907")
+end
```

Comment:
If you find yourself copying and pasting code or writing very similar code it is a good time to wonder whether you could be doing this in a loop.

In this case you could do:

```ruby
[
  ["A Clockwork Orange", "0393312836"],
  ["Crime and Punishment", "0679734503"],
  ["Transcending CSS", "0321410971"],
  ["The Very Hungry Caterpillar", "0399226907"]
].each do |name, isbn|
  unless Book.exists?(isbn: isbn)
    Book.create!(name: name, isbn: isbn)
  end
end
```

on another note, ActiveRecord has a method that you might like to use here:

```ruby
Book.find_or_create_by(name: name, isbn: isbn)
```

https://apidock.com/rails/v4.0.2/ActiveRecord/Relation/find_or_create_by

which will combine the `exists?` check and `create`